src/common.ts

```diff
@@ -81,7 +80,7 @@ export class AADMeta implements Datagram<AADMeta> {
   deserialize(data: Uint8Array): AADMeta | null {
     const buf = { bs: data };
     const metadataVersion = _decoder.decode(extractVarintPrefixed(buf));
-    if (metadataVersion !== AADMeta.METADATA_VERSION) {
+    if (metadataVersion !== AADMETA_VERSION) {
```

Comment:
this would actually make it incompatible with artifacts generated by the `0.1.0` release. I'll be modifying this momentarily.